lightning/src/ln/channelmanager.rs

```diff
@@ -6574,6 +6626,7 @@ impl_writeable_tlv_based_enum_upgradable!(PendingOutboundPayment,
 	(1, Fulfilled) => {
 		(0, session_privs, required),
 		(1, payment_hash, option),
+		(false, timer_ticks_without_htlcs, (reset_on_reload, 0)),
```

Comment:
Is it possible that mobile clients would never hit `IDEMPOTENCY_TIMEOUT_TICKS` if they never have the app open for 7 minutes+ straight? Maybe a block height check could still be a useful guardrail